packages/trader/src/Modules/SmartChart/Components/all-markers.jsx

```diff
@@ -492,6 +496,33 @@ const TickContract = RawMarkerMaker(
                 zoom: exit.zoom,
                 icon: ICONS.END.with_color(color, getColor({ status: 'bg', is_dark_theme })),
             });
+            if (is_accu_contract_ended) {
+                drawAccuBarrierRange({
```

Comment:
This forces barrier range to remain on DTrader chart for every recently closed contract.